src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -277,9 +283,15 @@ internal PackageManagerResult Undeprecate(string name)
 
         internal bool DoesCurrentUserOwnPackage(Package package,string username) 
         {
+            bool value;
+            if (this.packageMaintainers.Count > 0 && this.packageMaintainers.TryGetValue(package.Name, out value)) {
```

Comment:
If I remember correctly, you can use the following to save the definition? 
```
f (this.packageMaintainers.Count > 0 && this.packageMaintainers.TryGetValue(package.Name, out bool value))
```

Reply:
I had to use `value` below at line 293 so I used a variable for it.